repo/db/db.go

```diff
@@ -198,10 +198,10 @@ func initDatabaseTables(db *sql.DB, pin string) error {
 
     create table profile (key text primary key not null, value blob);
 
-    create table contacts (id text primary key not null, address text not null, username text not null, inboxes blob not null, added integer not null);
+    create table contacts (id text primary key not null, address text not null, username text not null, avatar text not null, inboxes blob not null, created integer not null, updated integer not null);
```

Comment:
again, pulled in from PubSub Contacts PR... this adds `avatar` and `updated` (used by search) to a `Contact`